contributing/dataset-ingestion/catalog-ingestion.qmd

```diff
@@ -184,7 +184,7 @@ if publish_response.ok:
 # should be available in the response of workflow execution request
 execution_id = "xxx"
 # url for execution status
-execution_status_url = f"{workflow_execution_url}/{execution_id}"
+execution_status_url = f"{STAGING_WORKFLOWS_API_URL}/discovery-executions/{execution_id}"
```

Comment:
Not tested, I'm not sure this does what we describe 
Most/all of our docs and maintained services are are _dataset_ workflows but this API is for _discovery_ workflows. Need to test this and either correct or remove from docs.